.gitignore

```diff
@@ -1,2 +1,2 @@
-**/.DS_Store
+.DS_Store
```

Comment:
Do we need to ignore tools/src either?

Reply:
While the `go.sum` files are a little obnoxious (perhaps we should hide them with `.gitattributes`, as in https://github.com/envoyproxy/gateway/pull/128), but it's my intent that `tools/src/` does get checked in; it's how all of the specific versions get pinned, since in general pinning all of those things is good.

Reply:
I see, thanks for sharing the details.